docker/Dockerfile.devito

```diff
@@ -4,14 +4,38 @@
 
 # Base image with compilers
 ARG base=devitocodes/bases:cpu-gcc
+ARG petsc=""
+
+##################  Install PETSc ############################################
+FROM $base as petsc_builder 
+
+ARG petsc
+
+RUN if [ "$petsc" = "true" ]; then \
+        apt-get update && apt-get install -y libopenblas-serial-dev git pkgconf mpich && \
+        python3 -m venv /venv && \
+        /venv/bin/pip install --no-cache-dir --upgrade pip && \
+        /venv/bin/pip install --no-cache-dir --no-binary numpy numpy && \
+        rm -rf ~/.cache/pip && \
+        mkdir -p /home/petsc && \
```

Comment:
No install it with the rest of the externals in `/opt/petsc` and it will be automatically copied through the copy `/opt`